objects/src/lib.rs

```diff
@@ -10,20 +10,20 @@ use assembly::{
 };
 use crypto::{
     hash::rpo::{Rpo256 as Hasher, RpoDigest as Digest},
-    merkle::{MerkleError, MerkleStore, Mmr, TieredSmt},
+    merkle::{MerkleError, Mmr, TieredSmt},
     utils::{
-        collections::{BTreeMap, Vec},
+        collections::Vec,
         string::{String, ToString},
     },
     Felt, StarkField, Word, WORD_SIZE, ZERO,
 };
 use vm_core::code_blocks::CodeBlock;
-use vm_processor::{AdviceInputs, StackOutputs};
+use vm_processor::AdviceInputs;
 
-mod accounts;
+pub mod accounts;
 pub use accounts::{
     validate_account_seed, Account, AccountCode, AccountDelta, AccountId, AccountStorage,
-    AccountStub, AccountType, AccountVault, StorageItem, DEFAULT_ACCOUNT_CODE,
+    AccountStub, AccountType, AccountVault, StorageItem,
 };
```

Comment:
Exporting both full account module as well as individual members from the module seems a bit redundant. I'd prefer to do either one or the other. Probably, a better approach is to export just the `accounts` module as it would be more consistent with how we export all other module from here.

Reply:
I did this because not all names are re-exported, for example, `AccountStorageDelta`, `get_account_seed`. In [past PR reviews](https://github.com/0xPolygonMiden/miden-base/pull/207#discussion_r1303690798) exporting new names was considered bad practice, so I did not add them here. Removing the exported names is a breaking change, and we really should not make a habit of breaking API (we have frequently broken upstream packages because of such things, for example, when changing the crypto repo)

We have discussed this a few times, but I'd like to point it again. I don't know what is the policy for name re-exports, and when writing code I have no idea what will be accepted or not in a PR review. This means I can't write a simple PR like this one, that only moves code around, without going over a lengthy PR review, for something that IMHO is not that important. Could we please write down some guidelines so we can streamline these simple things and remove them from PR review?

Reply:
remove re-exports

Reply:
> I did this because not all names are re-exported, for example, `AccountStorageDelta`, `get_account_seed`. In [past PR reviews](https://github.com/0xPolygonMiden/miden-base/pull/207#discussion_r1303690798) exporting new names was considered bad practice, so I did not add them here. Removing the exported names is a breaking change, and we really should not make a habit of breaking API (we have frequently broken upstream packages because of such things, for example, when changing the crypto repo)

I actually think that it's totally fine to introduce breaking changes at this point. Nobody is using these packages yet except for us, so - improving interface and APIs, even at the expense of breaking changes, is desirable so that we don't end up with confusing APIs later on when changing them will be much more difficult.

> We have discussed this a few times, but I'd like to point it again. I don't know what is the policy for name re-exports, and when writing code I have no idea what will be accepted or not in a PR review. This means I can't write a simple PR like this one, that only moves code around, without going over a lengthy PR review, for something that IMHO is not that important. Could we please write down some guidelines so we can streamline these simple things and remove them from PR review?

Yes, I can write down some common style guidelines. Will be a living list though as I probably won't capture everything on the first try.

For this PR specifically though, the length was not so much due to nit comments but because we needed to figure out where different things go. If there are only nits like this remain, I usually make comments and approve the PR (as I've done in this case too).